apps/invoke-groups/overlays/production/generated/deployment.yaml

```diff
@@ -46,7 +46,6 @@ spec:
           command:
             - node
             - --experimental-network-imports
-            - --experimental-vm-modules
```

Comment:
What is the benefit of removing the `--experimental-vm-modules`?

Reply:
Just a cleanup, I added it initially when was experimenting with vm, but it's not in use